packages/kbn-user-profile-components/src/user_profiles_selectable.tsx

```diff
@@ -340,12 +342,24 @@ export const UserProfilesSelectable = <Option extends UserProfileWithAvatar | nu
               gutterSize="s"
               responsive={false}
             >
-              <EuiFlexItem>
-                <EuiHighlight search={searchValue}>{option.label}</EuiHighlight>
+              <EuiFlexItem style={{ minWidth: 0, flexBasis: 'auto' }}>
+                <EuiHighlight
+                  css={css`
+                    ${euiTextTruncate()}
+                  `}
```

Comment:
Super quick notes about this:
- You can also use `className="eui-textTruncate"` if you prefer!
- If you prefer Emotion CSS, I'd suggest DRYing out/defining this CSS statically, e.g.
```
const truncateCss = css`${euiTextTruncate()}`;

...

<EuiHighlight css={truncateCSS} ...>
<EuiTextColor css={truncateCSS} ...>
```